tests/command_line/test_show_mask_info.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, division, print_function
+
+import procrunner
+
+
+def test_show_mask_info(dials_data):
+
+    data = dials_data("image_examples") / "dectris_eiger_master.h5"
+
+    result = procrunner.run(["dev.dxtbx.show_mask_info", data.strpath])
```

Comment:
With the suggestions above, you could just call the `dxtbx.command_line.show_mask_info.run()` (or `dxtbx.command_line.show_mask_info.show_mask_info()` method directly, without the need for the added `procrunner` level of indirection. If so, you would probably need to use the pytest `capsys` fixture to capture and examine the stdout:
https://docs.pytest.org/en/stable/capture.html#accessing-captured-output-from-a-test-function